internal/controller/api/v1alpha1/cluster_types.go

```diff
@@ -31,18 +31,18 @@ type ClusterSpec struct {
 	// Installation will start once cluster is in known state and approved
 	Approved                 bool   `json:"approved,omitempty"`
 	OpenshiftVersion         string `json:"openshiftVersion"`
-	BaseDnsDomain            string `json:"baseDnsDomain,omitempty"`
-	ClusterNetworkCidr       string `json:"clusterNetworkCidr,omitempty"`
+	BaseDNSDomain            string `json:"baseDNSDomain,omitempty"`
+	ClusterNetworkCidr       string `json:"clusterNetworkCIDR,omitempty"`
 	ClusterNetworkHostPrefix int64  `json:"clusterNetworkHostPrefix,omitempty"`
-	ServiceNetworkCidr       string `json:"serviceNetworkCidr,omitempty"`
-	ApiVip                   string `json:"apiVip,omitempty"`
-	ApiVipDnsName            string `json:"apiVipDnsName,omitempty"`
-	IngressVip               string `json:"ingresVip,omitempty"`
-	MachineNetworkCidr       string `json:"machineNetworkCidr,omitempty"`
-	SshPublicKey             string `json:"sshPublicKey,omitempty"`
-	VipDhcpAllocation        bool   `json:"vipDhcpAllocation,omitempty"`
-	HttpProxy                string `json:"httpProxy,omitempty"`
-	HttpsProxy               string `json:"httpsProxy,omitempty"`
+	ServiceNetworkCidr       string `json:"serviceNetworkCIDR,omitempty"`
+	APIVip                   string `json:"apiVIP,omitempty"`
+	APIVipDNSName            string `json:"apiVIPdnsName,omitempty"`
```

Comment:
@mhrivnak I'm not 100% sure `apiVIPdnsName` is valid, convention-wise. Let me know what you think.